doc/Manuals/ug-cs_x-road_6_central_server_user_guide.md

```diff
@@ -526,14 +526,15 @@ Note that in an HA setup, each node has its own set of configuration signing key
 The steps of key change are as follows:
 
 - First, a new key is generated (on each node in HA setups) and the configuration anchor containing the public key part(s) of the key(s) is distributed to X-Road participants. Until all participants have received the public key(s), the old (i.e. current) key(s) is/are used for signing configuration.
-- Then, after all participants have received and uploaded the new public key(s), the old key(s) is/are removed and the new key(s) is/are used to sign configuration.
+- Then, after all participants have received & applied the new public key(s), the old key(s) is/are removed and the new key(s) is/are used to sign configuration.
```

Comment:
The document version number and version history table should be updated.

Reply:
Good catch!